<launch>
  <arg name = "serial_ID" default= "agent_1"/>
  <arg name = "start_position"/>
  <node pkg="mz_wang_intern" type="agent" name="$(arg serial_ID)" output="screen" args="$(arg serial_ID)">
    <param name="serial_id" value="$(arg serial_ID)" />
    <rosparam param="start_position"  subst_value="true">$(arg start_position)</rosparam>
  </node>
</launch>
